viewpump/src/main/java/io/github/inflationx/viewpump/ReflectiveFallbackViewCreator.java

```diff
@@ -0,0 +1,52 @@
+package io.github.inflationx.viewpump;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.AttributeSet;
+import android.view.View;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+
+class ReflectiveFallbackViewCreator implements FallbackViewCreator {
+
+    private static final Class<?>[] constructorSignature2 = new Class[] {
+            Context.class,
+            AttributeSet.class
+    };
+
+    private static final Class<?>[] constructorSignature1 = new Class[] {
+            Context.class
+    };
+
+    @Nullable
+    @Override
+    public View onCreateView(@Nullable View parent, @NonNull String name, @NonNull Context context, @Nullable AttributeSet attrs) {
+        try {
+            Class<? extends View> clazz = Class.forName(name).asSubclass(View.class);
+            Constructor<? extends View> constructor;
+            Object[] constructorArgs;
+            try {
+                constructor = clazz.getConstructor(constructorSignature2);
```

Comment:
So I think the underlying `LayoutInflator` caches the constructors to save on looking them up every time for the same class, not sure how far we want to go with this, or... how often is this used over the layout inflator directly? I.e. are we still using the LayoutInflator and this is special case?